content/en/docs/tasks/observability/kiali/index.md

```diff
@@ -140,12 +136,10 @@ $ oc patch clusterrole kiali -p '[{"op":"add", "path":"/rules/-", "value":{"apiG
 1.  To open the Kiali UI, execute the following command in your Kubernetes environment:
 
     {{< text bash >}}
-    $ kubectl -n istio-system port-forward $(kubectl -n istio-system get pod -l app=kiali -o jsonpath='{.items[0].metadata.name}') 20001:20001
+    $ istioctl dashboard kiali
     {{< /text >}}
 
-1.  Visit <https://localhost:20001/kiali/console> in your web browser.
-
-1.  To log into the Kiali UI, go to the Kiali login screen and enter the username and passphrase stored in the Kiali secret.
+1.  To log into the Kiali UI, enter the username and passphrase stored in the Kiali secret.
```

Comment:
append `(demo/demo if you are using the demo profile)` ?

Reply:
It's `admin/admin` but I get the idea - I'll add that.

Reply:
ok, that is in..

```
1.  To log into the Kiali UI, enter the username and passphrase stored in the Kiali secret (`admin/admin` if you are using the demo profile).
```